tests/ci/common_posix_setup.sh

```diff
@@ -61,6 +68,38 @@ function build_and_test {
   run_cmake_custom_target 'run_tests'
 }
 
+function generate_symbols_file {
+  # read_symbols.go currently only support static libraries
+  if [ ! -f  "$BUILD_ROOT"/crypto/libcrypto.a ]; then
+    echo "Static library not found: ${BUILD_ROOT}/crypto/libcrypto.a"
+    print_system_and_dependency_information
+    exit 1
+  fi
+
+  go run "$SRC_ROOT"/util/read_symbols.go -out "$BUILD_ROOT"/symbols_crypto.txt "$BUILD_ROOT"/crypto/libcrypto.a
+  go run "$SRC_ROOT"/util/read_symbols.go -out "$BUILD_ROOT"/symbols_ssl.txt "$BUILD_ROOT"/ssl/libssl.a
```

Comment:
Nit: Is `libdecrepit.a` needed?

Reply:
It's not currently supported, but it's worth considering. We should revisit supporting the prefixing of symbols in `libdecrepit.a` once we make decisions regarding how we want to handle this library long-term.